tests/Http/RequestManagerTest.php

```diff
@@ -53,5 +53,9 @@ public function shouldSendAndDecodeRequest(): void
             $recordedRequests[0]->getBody()->__toString()
         );
         $this->assertSame(['application/json'], $recordedRequests[0]->getHeader('Content-Type'));
+        $this->assertStringStartsWith(
+            'php-client/',
```

Comment:
Why not assert the entire header value?

Reply:
Version will change every release. But ok, I can use the constant there, why not 🤷‍.